<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingingView</class>
 <widget class="QWidget" name="SingingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="viewLayout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="menuButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 127); 
background-color: rgb(187, 188, 255);</string>
         </property>
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 127); 
background-color: rgb(187, 188, 255);</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 127); 
background-color: rgb(187, 188, 255);</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 127); 
background-color: rgb(187, 188, 255);</string>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Repeat</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>40</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Start (sec)</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="repeatStartSpinBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QSpinBox" name="repeatEndSpinBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>70</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>End (sec)</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="repeatCheckBox">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>70</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>On/Off</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="volumeWidget" native="true"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Microphone Threshold:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QSlider" name="silentThresholdSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
